Documentation/TransferringFIles.md

```diff
@@ -0,0 +1,26 @@
+# Transferring files from Serenity to your host machine
+
+## Method 1: WebServer
+Serenity has a built-in web server which extends to your host machine.
+
+Open a new terminal and use the following command to start a WebServer instance for the home directory:
+
+ ```
+ ws .
+ ```
```

Comment:
Why is this indented by one space? Also should have a code block language (`console`).

Reply:
I didn't know `console` would be the correct code block language for this case, I'll change that.

As for the indentation, I generally feel it's easier to copy-paste if it's one line, but thinking about it now I don't know if the clipboard is generally carried over into QEMU. Either way, if this is not how Serenity documentation is written I am fine with that.

Reply:
Where are you seeing an intent? Do you mean because this is 3 lines instead of just one?

Reply:
No, I mean what I wrote - it's indented by one space:

![image](https://user-images.githubusercontent.com/19366641/148132398-647ab3c7-5b42-49ea-ae36-0dbe8f1a02b1.png)